src/Helper/ExportHelper.php

```diff
@@ -45,25 +48,16 @@ class ExportHelper extends AbstractExportHelper
      * @inheritdoc
      * @suppress PhanAccessMethodInternal, PhanUndeclaredClassConstant, PhanUndeclaredClassMethod
      */
-    public function encrypt($secret, $data)
+    public function encrypt(string $secret, $data)
     {
-        //Check if phpseclib v3 is used
-        //needed for comaptibility with Magento 2.4 versions
-        if (class_exists("phpseclib3\Crypt\AES")) {
```

Comment:
so, this selection between `phpseclib3` and `phpseclib` is not needed anymore? aka: assured to be there?